### DISCUSSION TITLE
reduce space between cventry

### DISCUSSION THREAD

**Discussion by HaoyangFan96** [posquit0/Awesome-CV#501] [Q&A] [answered]
I want to put multiple education backgrounds using cventries in the education section. However, currently the space between two cventry is quite large, which I would like to reduce the space between them. I will appreciate if anybody can provide me some hints on how to accomplish it.

**Comment by vedantpuri** [comment]
You could use the \vspace  command for this. Look it up online on how to use it. That's at least one of the solutions that may work for you

**Comment by ahmetuysal** [comment, marked as answer]
I also fixed this issue by adding `\vspace*{-0.5cm}` between each cv entry. in cv.tex. 
For example,
```
\vspace*{-0.5cm}
\input{cv/education.tex}
%\input{cv/skills.tex}
\vspace*{-0.5cm}
\input{cv/experience.tex}
```

**Comment by keivanipchihagh** [comment]
@ahmetuysal Great solution! Works like a charm.
